exercise_09/getDistanceMessageFromSumTo100.js

```diff
@@ -1,21 +1,18 @@
-/** 
+/**
  * #9 :: Export JS function "getDistanceMessageFromSumTo100" to generate a random number (in between 0-100) and sum it to given number
   - if sum result exceeds 100, then return message "Sum with value N exceeds in N from number 100"
   - otherwise, return message "Sum with value N is left in N from number 100"
  * input: number (value) 
  * output: string
  */
+function getDistanceMessageFromSumTo100(value) {
+    const randomNumber = Math.floor(Math.random() * 101);
+    const sum = value + randomNumber;
+    if (sum > 100) {
```

Comment:
Avoid magic numbers. 

```suggestion
    if (sum > THRESHOLD) {}
```

- [ ] Extract number to constant